apis/r/R/utils.R

```diff
@@ -110,6 +110,65 @@ uns_hint <- function(type = c('1d', '2d')) {
   }
 }
 
+.type_hint <- function(type) {
+  nm <- 'soma_r_type_hint'
```

Comment:
I think asking `nm` to be spelled out `name` is a reasonable ask

Reply:
`nm` is using the R standard name (see [`?setNames`](https://stat.ethz.ch/R-manual/R-devel/library/stats/html/setNames.html))

Reply:
I concede to the R world ✅